healthcheck/healthcheck.go

```diff
@@ -148,6 +150,12 @@ func checkHealth(serverURL *url.URL, backend *BackendHealthCheck) bool {
 	client := http.Client{
 		Timeout: backend.requestTimeout,
 	}
+	if backend.Options.IgnoreCertificateErrors {
+		tr := &http.Transport{
```

Comment:
I think it should be better to use the existing transport with TLS client configuration used for backend instead of add a new option to healthcheck.
https://github.com/containous/traefik/blob/be306d651e999feb50f1f126eb68073d607324f3/server/server.go#L68

Reply:
Ok. I'll pass that through